osmosis-1/osmosis.zone_assets.json

```diff
@@ -1566,6 +1566,13 @@
       "path": "transfer/channel-1411/umpwr",
       "osmosis_main": false,
       "osmosis_frontier": true
+    },
+    {
+      "chain_name": "juno",
+      "base_denom": "unexx",
```

Comment:
In this case I believe the base denom is: factory/juno1qly4zcmzr2gyxtze5yt9chv2srczwwunppxjfh/NEXX